docs/src/components/code-block/index.tsx

```diff
@@ -6,15 +6,24 @@ import { Language } from 'prism-react-renderer'
 import { ExamplesSwitcher, Example, parseCode } from './examples'
```

Comment:
```suggestion
import React from 'react'
import { ExamplesSwitcher, Example, parseCode } from './examples'
```

Reply:
this is still needed i think

Reply:
React is still imported. It's on line 3.